daft/execution/execution_step.py

```diff
@@ -352,10 +351,7 @@ def _handle_tabular_files_scan(
         filepaths_partition: Table,
     ) -> Table:
         data = filepaths_partition.to_pydict()
-        assert (
-            self.filepaths_column_name in data
-        ), f"TabularFilesScan should be ran on vPartitions with '{self.filepaths_column_name}' column"
-        filepaths = data[self.filepaths_column_name]
+        filepaths = data["file_paths"]
```

Comment:
Eh we weren't using a constant for `"file_sizes"` or `"num_rows"`, and I personally think that we can keep the contract informal for now since we're going to continue to rework this as (1) we move to the new query planner, and (2) we move file globbing to execution time.

If you feel strongly about this, I can try to expose the individual column names as fields on `FileInfo` and `FileInfos`, and propagate them as sidecar data whenever we pass around the `Table` representation.